boot.go

```diff
@@ -182,6 +182,9 @@ func (bs *Bootstrapper) Boot() int {
 	statusCode := 0
 	if !state.Wait(ctx) {
 		statusCode = 1
+		for _, err := range state.Errors() {
```

Comment:
I'm embarrassed, thank you :)

Reply:
Haha, no problem! This has been something that's actually frustrated me for a bit so I'm glad have an excuse to dig in.